12_findTheOldest/solution/findTheOldest-solution.js

```diff
@@ -9,11 +16,24 @@ const findTheOldest = function (array) {
   });
 };
 
-const getAge = function (birth, death) {
-  if (!death) {
-    death = new Date().getFullYear();
-  }
-  return death - birth;
-};
+// An alternative solution
+// const getAge = function (person) {
+//  // The nullish coalescing assignment operator
+//  // only does the assignment if the left side is "nullish" (evaluates to undefined or null)
+//  // if the left side has any other value, no assignment happens
+//  // here, we use ??= to set the current year for our subtraction below only if there is no year of death
+//  person.yearOfDeath ??= new Date().getFullYear();
+//
+//  return person.yearOfDeath - person.yearOfBirth;
+//};
+
+//const findTheOldest = function (people) {
+//  const peopleOldestToYoungest = people.toSorted(
+//    (person, nextPerson) => getAge(nextPerson) - getAge(person),
+//  );
+//
+//  const oldestPerson = peopleOldestToYoungest[0];
+//  return oldestPerson;
+//};
```

Comment:
Another nit solely for the sake of readability. Lots of `//` really gets in the way :sweat_smile: 

```suggestion
/* ALTERNATIVE SOLUTION
const getAge = function (person) {
  // The nullish coalescing assignment operator
  // only does the assignment if the left side is "nullish" (evaluates to undefined or null)
  // if the left side has any other value, no assignment happens
  // here, we use ??= to set the current year for our subtraction below only if there is no year of death
  person.yearOfDeath ??= new Date().getFullYear();

  return person.yearOfDeath - person.yearOfBirth;
};

const findTheOldest = function (people) {
  const peopleOldestToYoungest = people.toSorted(
    (person, nextPerson) => getAge(nextPerson) - getAge(person),
  );

  const oldestPerson = peopleOldestToYoungest[0];
  return oldestPerson;
};
*/
```